doc/source/development/maintaining.rst

```diff
@@ -90,15 +93,19 @@ Here's a typical workflow for triaging a newly opened issue.
    If a reproducible example is provided, but you see a simplification,
    edit the original post with your simpler reproducible example.
 
+   Ensure the issue exists on the main branch and that it has the "Needs Triage" tag
+   until all steps have been completed.
```

Comment:
I think it'd be helpful to report your action in the issue here so we can avoid duplicating effort. What do you think about something like "Once you have confirmed the issue exists on main, make a comment in the issue stating this so that others know it's been confirmed."

Reply:
Sounds good to me! I'll add that now.